modules/ssh/ssh.go

```diff
@@ -211,7 +212,11 @@ func Listen(host string, port int, ciphers []string, keyExchanges []string, macs
 	}
 
 	keyPath := filepath.Join(setting.AppDataPath, "ssh/gogs.rsa")
-	if !com.IsExist(keyPath) {
+	isExist, err := util.IsExist(keyPath)
+	if err != nil {
+		log.Fatal("Unable to check if %s exists. Error: %v", keyPath, err)
+	}
+	if !isExist {
```

Comment:
```if err == nil && !isExist {```

Reply:
Not necessary as the fatal will kill server first

Reply:
If ```log.Fatal``` means: log + exit then ok.